src/web_peer_connection.cpp

```diff
@@ -640,7 +643,7 @@ void web_peer_connection::handle_redirect(int const bytes_left)
 
 		// add_web_seed won't add duplicates. If we have already added an entry
 		// with this URL, we'll get back the existing entry
-		web_seed_t* web = t->add_web_seed(redirect_base, web_seed_entry::url_seed, m_external_auth, m_extra_headers);
+		web_seed_t* web = t->add_web_seed(redirect_base, web_seed_entry::url_seed, m_external_auth, m_extra_headers, true);
```

Comment:
redirected web_seed should not be stored in resume because it is only serialize url part (without "redirects" map") e.g original url points to "http://example1.com/file1", web server response with location "http://example2.com/subpath/file2". (m_url=="http://example2.com/", redirects[0]=="/subpath/file2"). if we try to load old save resume then library tryies to request wrong path "http://example2.com/file1" ("redirects" map is lost).